components/raftstore/src/store/worker/read.rs

```diff
@@ -62,44 +63,40 @@ impl ReadDelegate {
         self.invalid.store(true, Ordering::Release);
     }
 
+    pub fn fresh_valid_ts(&mut self) {
+        self.last_valid_ts = monotonic_raw_now();
+        // We add one second to make sure that snapshot-cache sent by other thread is really created after this region has applied to current term exactly
+        self.last_valid_ts.sec += 1;
```

Comment:
Why does it need to add 1? It seems the `monotonic_raw_now()` is enough.

Reply:
Because there may be a slight error in the time obtained from different threads